src/backend/common/CommandBuffer.cpp

```diff
@@ -476,6 +584,15 @@ namespace backend {
                             return false;
                         }
 
+                        auto& subpassInfo = currentRenderPass->GetSubpassInfo(currentSubpass);
```

Comment:
In the object introduced to handle all the usage tracking, this could be done with a "temporarily frozen" tag put on resources.